aries-site/src/pages/templates/wizard.mdx

```diff
@@ -24,10 +25,31 @@ import {
   <WizardValidationExample />
 </Example>
 
-## Guidance
+## When to use
 
 Use a wizard when your form involves multiple steps. This makes the experience more manageable for the user and helps to eliminate error by validating inputs as the user progresses.
 
+## Anatomy
+
+<Example plain>
+  <WizardAnatomy />
+</Example>
+
+1. **Wizard header**: This region is intended to provide user with context on what the wizard is about.
+   - **a. Wizard title** (required): A short and descriptive title that identifies the wizard.
+   - **b. Previous step button** (required): A button that allows the user to move back one step in the wizard. This button is aligned to the left side of the header and is present on all steps expect the first.
+   - **c. Cancel button** (required): A button that allows the user to leave the wizard. When clicked, a confirmation modal will appear. This button is aligned to the right side of the header.
+1. **Step identifier:** This region is intended to provide the user with context on what the step is about.
+   - **a. Step title** (required): A short and descriptive title that uniquely identifies the step.
+   - **b. Step summary** (optional): A summary of where the user is within the wizard, formatted as "Step X of Y".
+   - **c. Step description** (optional): Any additional context or information about the given step.
+1. **Step inputs**: Form inputs for the current step.
+1. **Step footer**: The footer contains a primary action button to advance to the next step or finish the wizard. The footer should remain a consistent width across all steps of a wizard. Its width should match the width of the widest step.
+   - **a. Next/finish button** (required): A primary action button that allows the user to advance to the next step or finish the wizard. This button should be aligned to the far-right side of the footer.
+1. **Cancel wizard modal** (required): A center modal that provides double confirmation before the wizard is exited and cancelled. It appears when the user clicks on the "Cancel" button.
```

Comment:
Should we mention something about "not displayed?"

Should we eventually add an anatomy diagram for the "cancel wizard modal?"

Reply:
I'll mention "Not displayed" for now and we can handle the added anatomy diagram separately.

Reply:
Incorporated in latest commit.